spec/factories/payment_responses.rb

```diff
@@ -0,0 +1,86 @@
+FactoryBot.define do
+  sequence :adyen_payments_response do
+    OpenStruct.new(
+      response: {
+        "additionalData" => {
+            "recurringProcessingModel" => "UnscheduledCardOnFile"
+        },
+        "pspReference" => "#{SecureRandom.uuid}",
+        "resultCode" => "Authorised",
+        "amount" => {
+            "currency" => "USD",
+            "value" => 1000
+        },
+        "merchantReference" => "#{SecureRandom.uuid}",
+        "paymentMethod" => {
+            "brand" => "amex",
+            "type" => "scheme"
+        }
+      }
+    )
+  end
+
+  sequence :adyen_payment_links_response do
+    OpenStruct.new(
+      response: {
+        "amount" => {
+            "currency" => "EUR",
```

Comment:
Layout/FirstHashElementIndentation: Use 2 spaces for indentation in a hash, relative to the parent hash key.